extensions/src/ACRE2Arma/arma2ts/CallExt_DllMain.cpp

```diff
@@ -9,23 +9,34 @@
 #include <algorithm>
 #include "shlobj.h"
 #include "Shlwapi.h"
-#include <algorithm> 
-#include <functional> 
+#include <algorithm>
+#include <functional>
 #include <cctype>
 #include <locale>
 
+#include <winsock2.h>
+#include <ws2tcpip.h>
+#include <mstcpip.h>
+#include "wine.h"
+
 #pragma comment(lib, "shlwapi.lib")
+#pragma comment(lib, "Ws2_32.lib")
 
 #define PIPE_COMMAND_OPEN    0
 #define PIPE_COMMAND_CLOSE    1
 #define PIPE_COMMAND_WRITE    2
 #define PIPE_COMMAND_READ    3
 #define PIPE_COMMAND_RESET    4
+#define COMMAND_IS_WINE 99
+
+#define WINE_SOCKET_PORT "19141"
```

Comment:
Arbitrary; not on the https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers#Well-known_ports